.storybook/preview-head.html

```diff
@@ -7,4 +7,16 @@
   .space-docs {
     margin-bottom: 30px;
   }
+
+  .docs {
+    padding: 12px;
```

Comment:
Could we use the [niten tokens](https://github.com/getninjas/niten-tokens/blob/master/build/web/_niten-tokens.scss) in storybook docs?

Reply:
No, it's Style is specified to Storybook iframe and not use Niten.